docs/workforce-management-framework/topics/pro-Using-Keycloak-implementation.adoc

```diff
@@ -0,0 +1,89 @@
+
+NOTE: If you wish to configure your own instance of {WFM-RC-KeycloakName} see the following
+link:{WFM-RC-KeycloakURL}getting_started/topics/first-boot.html[getting started guide]. You will need to import our Realm
+settings by importing link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/data_files/raincatcher-realm.json[/data_files/raincatcher-realm.json].
+
+[[enabling-the-keycloak-demo-server]]
+== Enabling {WFM-RC-KeycloakName} on the Demo Server
+
+The following steps are for using the pre-configured docker image but will work for any instance of {WFM-RC-KeycloakName}
+provided you implement our Realm settings.
+
+- login into the {WFM-RC-KeycloakName} server via a browser using the following URL
+
+    http://localhost:8080/auth/
+
+- Click on `client` on the left menu
+- Select raincatcher-cloud from the list
+- Select `installation` from the selection bar at the top of the screen
+- From the `Format Options` drop down select `Keycloak OIDC JSON`
+- JSON Data similar to below will render (this data depends on the configuration of the {WFM-RC-KeycloakName} server and can vary)
+
+    {
+    "realm": "raincatcher",
+    "bearer-only": true,
+    "auth-server-url": "http://localhost:8080/auth",
+    "ssl-required": "external",
+    "resource": "raincatcher-cloud",
+    "use-resource-role-mappings": true
+    }
+
+- Create a new field `"keycloakConfig":` with the data copied from the previous step
+- Add this object to your production or development environment by editing the relevant file in the Demo Application
+
+link:{WFM-RC-CoreURL}{WFM-RC-Branch}/demo/server/config-dev.json[/demo/server/config-dev.json]
+
+link:{WFM-RC-CoreURL}{WFM-RC-Branch}/demo/server/config-prod.json[/demo/server/config-prod.json]
+
+Note: There is partially implemented of the `keycloakConfig` object at `/demo/server/config-prod.json` as an example.
+
+- Some small changes to code the mobile and and portal applications are required to disable passportAuth and enable {WFM-RC-KeycloakName}
+
+Change the following file link:{WFM-RC-AngularJsURL}{WFM-RC-Branch}/demo/mobile/src/app/app.js[/demo/mobile/src/app/app.js]
+[source,javascript]
+----
+    // Comment to disable passport auth
+    require('@raincatcher/demo-auth-passport')('wfm-mobile'),
+    // Uncomment to enable Keycloak
+    // require('../keycloak');
+----
+As above change the following file link:{WFM-RC-AngularJsURL}{WFM-RC-Branch}/demo/portal/src/app/main.js[/demo/portal/src/app/main.js]
+[source,javascript]
+----
+    // Comment to disable passport auth
+    require('@raincatcher/demo-auth-passport')('app'),
+    // Uncomment to enable Keycloak
+    // require('./keycloak');
+----
+
+- Routes will now be protected with {WFM-RC-KeycloakName}
+
+== Reference {WFM-RC-KeycloakName}
+
+To login into {WFM-RC-KeycloakName} Docker image navigate to http://localhost:8080/auth/ login as the admin user using the following credentials:
```

Comment:
"To login into {WFM-RC-KeycloakName} Docker image navigate"

**>**

"To login into `the` {WFM-RC-KeycloakName} `Admin Console,` navigate to http://localhost:8080/auth/ `and` login as the admin user using the following credentials:"

Reply:
:+1: